scripts/utils/tc_helper.py

```diff
@@ -275,3 +277,64 @@ def print_tc_helper(teammates_collection):
             for teammates in teammates_c:
                 for agent in teammates:
                     print(f'\t{agent.name}, score for layout {layout_name} is: {agent.layout_scores[layout_name]}, len: {len(teammates)}')
+
+def generate_TC_for_Adversary(args, 
+                            agent,
+                            train_types = [TeamType.HIGH_FIRST],
```

Comment:
why high_first?

Reply:
In most experiments, we assume we let out adversary agents to play with high-performance agents.